src/@context/Pool/index.tsx

```diff
@@ -68,7 +69,9 @@ function PoolProvider({ children }: { children: ReactNode }): ReactElement {
     setPoolData(response.poolData)
     setPoolInfoUser((prevState) => ({
       ...prevState,
-      poolShares: response.poolDataUser?.shares[0]?.shares
+      poolShares: response.poolDataUser?.shares[0]
```

Comment:
I think `poolShares: response?.poolDataUser?.shares[0]?.shares || '0'` does the same but shorter